### ISSUE TITLE
Tests fail with WebOb 1.6

### ISSUE THREAD

**Issue by Themanwithoutaplan** [Pylons/substanced#266]
WebOb now checks whether control characters are on contained. This leads to seven test failures like this.

Workaround would be to pin Webob at < 1.6a (thanks Zart) but obviously the mocking code needs fixing.

```
self = <substanced.folder.tests.test_views.TestFolderContents testMethod=test_duplicate_multiple>
mock_rename_duplicated_resource = <MagicMock name='rename_duplicated_resource' id='4396383200'>

    @mock.patch('substanced.folder.views.rename_duplicated_resource')
    def test_duplicate_multiple(self, mock_rename_duplicated_resource):
        context = mock.Mock()
        request = mock.Mock()
        request.localizer = testing.DummyRequest().localizer
        request.view_name = 'contents'
        request.params = {}
        request.POST.get.return_value = 'a/b'
        mock_rename_duplicated_resource.side_effect = ['a-1', 'b-1']

        inst = self._makeOne(context, request)
>       inst.duplicate()

substanced/folder/tests/test_views.py:844:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
substanced/folder/views.py:784: in duplicate
    return self.get_redirect_response()
substanced/folder/views.py:383: in get_redirect_response
    _query=qs)
lib/python3.4/site-packages/pyramid/httpexceptions.py:459: in __init__
    body_template=body_template, location=location, **kw)
lib/python3.4/site-packages/pyramid/httpexceptions.py:219: in __init__
    Response.__init__(self, status=status, **kw)
lib/python3.4/site-packages/webob/response.py:153: in __init__
    setattr(self, name, value)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

r = <HTTPFound at 0x1060b99a8 302 Found>, value = <Mock name='mock.sdiapi.mgmt_path()' id='4396367264'>

    def fset(r, value):
        fdel(r)
        if value is not None:
>           if '\n' in value or '\r' in value:
E           TypeError: argument of type 'Mock' is not iterable
```

**Comment by Themanwithoutaplan** [comment]
Zart has suggested a fix https://gist.github.com/zart/7f04d20814ba6c42b541

**Comment by stevepiercy** [comment]
Fixed by #269
